android/src/main/java/io/invertase/googlemobileads/common/ReactNativeAdView.java

```diff
@@ -13,9 +13,11 @@ public class ReactNativeAdView extends ReactViewGroup {
   private boolean manualImpressionsEnabled;
   private boolean propsChanged;
   private boolean isFluid;
+  private ThemedReactContext reactContext;
 
-  public ReactNativeAdView(final Context context) {
-    super(context);
+  public ReactNativeAdView(final ThemedReactContext context) {
+    super(context.getCurrentActivity());
```

Comment:
using the activity context here fixes the LogCat info message `I/Ads: Can not create dialog without Activity Context` & also makes the gesture work